### PR TITLE
Feat/Adjust Response and Request to be Messages

### PR DESCRIPTION
**Pull request by tom-rm-meyer-ISST**
While preparing the response api base, I realized that Response and Request must be Messages. Therefore I adjusted this across the different layers.

I also prepared a base controller with a service providing the consumption base for the respective API.

### CODE REVIEW

File: src/main/java/org/eclipse/tractusx/puris/backend/common/api/logic/dto/SuccessfullRequestDto.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package org.eclipse.tractusx.puris.backend.common.api.logic.dto;
```
Comment (mhellmeier) [inline src/main/java/org/eclipse/tractusx/puris/backend/common/api/logic/dto/SuccessfullRequestDto.java:1]: Please add a license header to every file.

**Review by mhellmeier — CHANGES_REQUESTED** [review]

File: src/main/java/org/eclipse/tractusx/puris/backend/common/api/logic/dto/SuccessfullRequestDto.java (hunk 1 above)
Comment (tom-rm-meyer-ISST) [reply src/main/java/org/eclipse/tractusx/puris/backend/common/api/logic/dto/SuccessfullRequestDto.java:21]: Added. Checked other files, too.

**Review by mhellmeier — CHANGES_REQUESTED** [review]
Please check conflicting files and possible duplicate commits due to the current status of the `main` branch.

**Comment by tom-rm-meyer-ISST** [comment]
> Please check conflicting files and possible duplicate commits due to the current status of the `main` branch.

Sorry wasn't aware of the workflow here. I created a new PR #35 to cleanly do the changes.

**Pull request closed without merging**